CHANGELOG.md

```diff
@@ -68,6 +68,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1695](https://github.com/open-telemetry/opentelemetry-python/pull/1695))
 - Change Jaeger exporters to obtain service.name from span
   ([#1703](https://github.com/open-telemetry/opentelemetry-python/pull/1703))
+- Split Zipkin exporter into `opentelemetry-exporter-zipkin-json` and
+  `opentelemetry-exporter-zipkin-proto` packages to reduce dependencies. The
```

Comment:
Should we be more explicit in package naming `opentelemetry-exporter-zipkin-proto-http`?

Reply:
Updated. I'll update the jaeger proto package in a separate PR to follow the same pattern `opentelemetry-exporter-jaeger-proto-grpc`.

I was wondering if it makes sense to add a protocol to the end of the other packages as well ie. `opentelemetry-exporter-zipkin-json` `opentelemetry-exporter-jaeger-thrift`? And should the jaeger package be split into a UDP and HTTP package?

In my opinion the big dependencies users would want to to avoid are protos and grpc, in which case i would vote to leave the other packages as is.

Reply:
> In my opinion the big dependencies users would want to to avoid are protos and grpc, in which case i would vote to leave the other packages as is.

Agree. I don't think there isn't much we gain from splitting them further.

Reply:
+1 on leaving packages as is.